client/state/analytics/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ANALYTICS_CONTINOUS_MONITOR_ON,
+	ANALYTICS_CONTINOUS_MONITOR_OFF
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const continuousTracking = createReducer( 'CHECKING', {
+	[ ANALYTICS_CONTINOUS_MONITOR_ON ]: () => 'TRACKING',
+	[ ANALYTICS_CONTINOUS_MONITOR_OFF ]: () => 'NOT_TRACKING'
+} );
```

Comment:
To make your reducer part of the Redux state tree, you add it to [this list](https://github.com/Automattic/wp-calypso/blob/8209735f95ae14e7f2e1bfb4ac19d6bfb9efee6d/client/state/index.js#L71-L121).